popSimManu.tex

```diff
@@ -941,13 +946,60 @@ \section*{Acknowledgments}
 \section*{Supplemental Figures}
 \begin{figure}
 \begin{center}
-\includegraphics[width=0.8\linewidth]{display_items/homo_sapiens_mask_Gutenkunst.png}
+\includegraphics[width=0.8\linewidth]{display_items/HomSap_AE9K19_chr22_map_AI.pdf}
+\caption{\textbf{Validating the \texttt{SLiM} engine backend under a genetic map}.
+Here we validate our integration of the \texttt{SLiM} \citep{haller2019tree,haller2019slim} engine backend. 
+We show quantile-quantile plots between \texttt{SLiM} and \texttt{msprime} engines for three 
+population genetic summary statistics: \textit{r}\textsuperscript{2}, Tajimas's pi, and Tajima's D. Additionally, we 
```

Comment:
Forgive my ignorance, but what is `r^2` here? Is that an average calculated from all pairs of sites? Just those with a certain separation distance?

Reply:
Thanks for asking about this. Here r^2 is the average calculated from all pairs of sites, but only for the first Mb, as I was initially running out of memory when binning by separation distance with the entire chromosome. I might not have the same memory issues if I'm not binning, so maybe I should recalculate the average for the entire chromosome? thoughts?

Reply:
With respect to your question about why the plots look different at Q=50, I not sure if we know for sure at the moment. Perhaps this would be worth speculating on in the discussion, and other's could further edit with their thoughts on the matter?

Reply:
I don't think it's necessary to change the r^2 calculation, as long as it's clear how it was calculated. If you based if off the code I included in the `validiation.py` script, which uses scikit-allel, then I'm pretty sure the memory issues would have been caused by `allel.rogers_huff_r()`, and not the binning. `allel.rogers_huff_r()` returns an array containing one `r` value for each pair of sites. Even using a conservative estimate of one variant per 1000 bp, that's 51304 variants for chr22, and about 10 GB of ram!

Reply:
And, yes, there probably needs to be some mention of the impact of scaling. But this seems to be a complex issue. There was some discussion of one particular aspect of scaling over in https://github.com/popsim-consortium/stdpopsim/issues/444, but I don't think we figured it out fully. Certainly, I could see there being room for a follow up paper on this alone. (If anyone's interested, feel free to ping me).